### PR TITLE
Add new slang

### PR DESCRIPTION
**Pull request by gukiboy**
## Pre-Submission Checklist

- You have only one commit (if not, squash them into one commit).
- All new and existing tests pass the command `yarn test` and `yarn lint`.

### CODE REVIEW

File: dialects/paulistes.json [hunk 1]
```diff
@@ -200,6 +200,15 @@
       "Descolei um emprego novo, começarei amanhã",
       "Descola aquela grana para mim"
       ]
+  },
+  {
+    "dialect": "MC (Como em MC Donalds)",
```
Comment (mvfsillva) [inline dialects/paulistes.json]: Hi, first, thanks for contributing :]

The phrase in parentheses `(Como em MC Donalds)` is it part of the dialect?

Comment (mvfsillva) [reply dialects/paulistes.json]: What do you think of using synonyms? 

`"synonyms": "MC Donalds"`

**Comment by mvfsillva** [comment]
@gukiboy  could you resolve the conflict?

File: dialects/paulistes.json (hunk 1 above)
Comment (gukiboy) [reply dialects/paulistes.json]: I thought about explaining as a meaning, it would look like:
"meanings": "Deve ser lido como o 'Mc' em "McDonalds"

**Pull request closed without merging**